src/pages/Home/components/HeaderLogo/HeaderLogo.js

```diff
@@ -1,12 +1,17 @@
-import React from 'react'
-
+import React from "react";
+import { useTheme } from "@mui/material/styles";
 
 const HeaderLogo = () => {
-  return (
-    <>
-      RIFFIN  
-    </>
-  );
-}
-
-export default HeaderLogo
\ No newline at end of file
+  const theme = useTheme();
+
+  const logoStyle = {
+    fontSize: theme.typography.h1.fontSize,
+    fontWeight: theme.typography.h1.fontWeight,
+    padding: "30px",
+    margin: "10px",
+  };
+
+  return <div style={logoStyle}>Riffin</div>;
```

Comment:
Cool! Tho I think it might need the div for importing the style here, as there is some spacial relationship issues with the logo during screen resizing.

Reply:
My bad, didn't realize you put the div in! This is good for now. Eventually a logo image will sit next to RIFFIN and on small breakpoints just the logo will show.